Homeworks/Homework_1/task_1.py

```diff
@@ -0,0 +1,19 @@
+# Вывести в консоль таблицу умножения от 2х2 до 9х10 как на школьной тетрадке.
+
+def maths(n, m):
+    for i in range(2, 11):
+        for j in range(n, m):
+            print(f'{j} * {i}', end='')
+            if i < 10:
+                print(f' = ', end='')
+            else:
+                print(f'= ', end='')
+            if j * i < 10:
+                print(f' {j * i}', end='            ')
```

Comment:
Так будет чуть лучше, на мой взгляд
```suggestion
                print(f' {j * i}', end='\t\t')
```